### DISCUSSION TITLE
Continues Release / Deployment (based on Conventional Commits)

### DISCUSSION THREAD

**Discussion by genofire** [kube-logging/logging-operator#1306] [General]
**Is your feature request related to a problem? Please describe.**
Some fixes needed long to go to the next releases:
 - #1278
 - #1244 
 - #1241

**Describe the solution you'd like**
Using semantic-release (bot) to auto release on every [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)):
https://semantic-release.gitbook.io/semantic-release/

**Describe alternatives you've considered**
release more often

**Additional context**
i normaly develop on gitlab - so no idea how to setup github-actions.

**Comment by aslafy-z** [comment]
I do this using GitHub actions for several of my projects. Since this is not a nodejs based repository, I recommend using a prepackaged action like https://github.com/marketplace/actions/action-for-semantic-release.

**Comment by pepov** [comment]
I like the idea, but I would prefer talking just a little bit about that on our discord or we can even schedule a call and go through what that would mean exactly.

https://discord.gg/b3JG2pBs

**Comment by genofire** [comment]
for semantic-release exists a helmchart plugin: that increase and release a new version also automaticaly.

- https://github.com/eshepelyuk/semantic-release-helm-oci
- https://github.com/m1pl/semantic-release-helm
- https://github.com/nflaig/semantic-release-helm

**Comment by csatib02** [comment]
At the moment we are releasing on a 6 week schedule, with approximately 1 major update, and 5-10 bugfixes per release. This schedule gives us enough time to carefully plan ahead with our roadmap. On the other hand, we do release the project's docker image and helm-chart after every merged PR, but it may contain issues.
